test/e2e-olm/elasticsearch_test.go

```diff
@@ -213,6 +220,46 @@ func elasticsearchFullClusterTest(t *testing.T, f *framework.Framework, ctx *fra
 		}
 	}
 
+	// Update the resource spec for the cluster
+	memValue := resource.MustParse("2Gi")
+	cpuValue := resource.MustParse("100m")
+	exampleElasticsearch.Spec.Spec.Resources = v1.ResourceRequirements{
+		Limits: v1.ResourceList{
+			v1.ResourceMemory: memValue,
+		},
+		Requests: v1.ResourceList{
+			v1.ResourceCPU:    cpuValue,
+			v1.ResourceMemory: memValue,
+		},
+	}
+
+	err = f.Client.Update(goctx.TODO(), exampleElasticsearch)
```

Comment:
I generally find this test frustrating because it's not obvious when it fails where it failed.  Consider adding Info steps which define what is about to happen